info/Community/mastodon.md

```diff
@@ -9,4 +9,16 @@ We run a Mastodon instance on [social.lol](https://social.lol), and all omg.lol
 
 Mastodon is kind of like Twitter, only much nicer. For a much better description, [read all about Mastodon here](https://joinmastodon.org).
 
-You can follow us on Mastodon at [@omgdotlol@social.lol](https://social.lol/@omgdotlol).
\ No newline at end of file
+You can follow us on Mastodon at [@omgdotlol@social.lol](https://social.lol/@omgdotlol).
+
+## If your address expires
+
+In the event of your address expiring, we will give you a window to move your Mastodon account and export your data.
```

Comment:
```suggestion
In the event of your address expiring, we’ll give you a window to move your Mastodon account and export your data.
```
"we’ll" seems to be used most of the time, i'm not adam thoufh
https://github.com/neatnik/omg.lol/search?q=%22we%27ll%22
https://github.com/neatnik/omg.lol/search?q=%22we+will%22